linalg/solvers.cpp

```diff
@@ -3938,25 +3907,17 @@ void NNLSSolver::Solve(const Vector& rhs_lb, const Vector& rhs_ub,
             lwork = -1;
             work.resize(10);
 
-#ifdef MFEM_USE_SINGLE
-            sormqr_(&lside, &trans, &m, &n_update, &i_qr_start,
-#elif defined MFEM_USE_DOUBLE
-            dormqr_(&lside, &trans, &m, &n_update, &i_qr_start,
-#endif
-                    mat_qr_data.GetData(), &m, tau.GetData(),
-                    mat_qr_data.GetData() + (i_qr_start * m), &m,
-                    work.data(), &lwork, &info);
+            MFEM_LAPACK_PREFIX(ormqr_)(&lside, &trans, &m, &n_update, &i_qr_start,
+                                       mat_qr_data.GetData(), &m, tau.GetData(),
+                                       mat_qr_data.GetData() + (i_qr_start * m), &m,
```

Comment:
Slightly exceeding 80 columns here and below.

Reply:
Fixed in e4b8584a16616f2b50422b85c5ba0fa30cd68c46